### PR TITLE
Replace local MongoDB role with external Ansible Galaxy dependency

### PR DESCRIPTION
**Pull request by armab**
Implements #9

Use **`Stouts.mongodb`** https://galaxy.ansible.com/list#/roles/982 as a dependency.

It installs MongoDB via recommended way: http://docs.mongodb.org/manual/tutorial/install-mongodb-on-ubuntu/ and also has basic TravisCI tests.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -28,6 +28,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.provision :hostmanager
   end
 
+  # Ensure role dependencies are in place
+  if ['up', 'provision'].include?(ARGV.first) && !(File.directory?('roles/Stouts.mongodb'))
```
Comment (jfryman) [inline Vagrantfile]: Is the check for role dependencies going to grow? This statement could get a bit gnarly, or there could be many of them copy/pasted. 

In what other ways might this be accomplished?

**Comment by jfryman** [comment]
Nice. Big fan of upstream consumption.

File: Vagrantfile (hunk 1 above)
Comment (armab) [inline Vagrantfile]: Yeah, don't like it either. For now it's simple workaround to see how 3rd party `mongo` role works.

* One of the solutions is using 3rd party Vagrant plugin [vagrant-host-shell plugin](https://github.com/phinze/vagrant-host-shell)
As a result installing Ansible Galaxy dependencies will be part of provision, which is more clear (`File.directory` will be removed too).
* Another solution is to create another Ansible playbook to install dependencies.

It's pretty strange that Ansible can't download dependencies automatically.
BTW, good point, thanks for taking a look :+1:

**Comment by armab** [comment]
Me too, sadly 3rd party Ansible roles are not so good as Puppet or Chef recipes.

Even the most popular roles usually support no more than 1-2 Linux distros.

**Comment by armab** [comment]
Closing, since external Galaxy roles dont't support all platforms we need.

**Pull request closed without merging**